src/test/java/com/epam/tc/hw2/ex2/tests/EpamPageTitleTest.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.tc.hw2.ex2.tests;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import com.epam.tc.hw2.ex2.GeneralWebDriver;
+import org.testng.annotations.Test;
+
+public class EpamPageTitleTest extends GeneralWebDriver {
```

Comment:
why it is a separated test?

Reply:
Because I created test classes for each step in the exercise.